### PR TITLE
Minor improvements to simplejson stubs

### PR DESCRIPTION
**Pull request by ilevkivskyi**

### CODE REVIEW

File: third_party/2/simplejson/scanner.pyi [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from typing import Any, IO
 
 class JSONDecodeError(ValueError):
+    doc: str = ...
```
Comment (JelleZijlstra) [inline third_party/2/simplejson/scanner.pyi]: Why did you add `doc` specifically? There are a few other attributes (msg, pos, end, lineno, colno, endlineno, endcolno).

Also, the methods listed below don't seem to exist.

Comment (ilevkivskyi) [reply third_party/2/simplejson/scanner.pyi]: I just stumbled at this error mypy was complaining about. But maybe you are right, it makes sense to add a bit more cleanup here.

File: third_party/2/simplejson/__init__.pyi [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 from simplejson.scanner import JSONDecodeError as JSONDecodeError
 from simplejson.decoder import JSONDecoder as JSONDecoder
 from simplejson.encoder import JSONEncoder as JSONEncoder, JSONEncoderForHTML as JSONEncoderForHTML
 
 def dumps(obj: Any, *args: Any, **kwds: Any) -> str: ...
 def dump(obj: Any, fp: IO[str], *args: Any, **kwds: Any) -> None: ...
-def loads(s: str, **kwds: Any) -> Any: ...
-def load(fp: IO[str]) -> Any: ...
+def loads(s: Text, **kwds: Any) -> Any: ...
+def load(fp: IO[str], **kwds: Any) -> Any: ...
```
Comment (JelleZijlstra) [inline third_party/2/simplejson/__init__.pyi]: We could specify the actual arguments, but that would be a lot more work, so **kwds is fine for now.
